src/MongoDB.Analyzer/Core/Poco/Data/Airport.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+    "airport": [
+        "Hartsfield-Jackson Atlanta International Airport",
```

Comment:
Maybe switch to airport codes, for shorter results?

Reply:
Added AirportCodes as well